scripts/TenantSetup/TenantHttpData/education/educationSchool.json

```
@@ -0,0 +1,22 @@
+{
+    "@odata.type": "#microsoft.graph.educationSchool",
+    "displayName": "Raptor created demo school",
+    "description": "String",
+    "externalSourceDetail": "system",
+    "principalEmail": "raptor@developers.devX.com",
```

nit: to prevent accidentally listing a real email that belongs to a third party, `contoso.com` is the default domain Microsoft uses in these scenarios.
```suggestion
    "principalEmail": "raptor@contoso.com",
```